code/datums/traits/good.dm

```diff
@@ -96,6 +96,14 @@
 	)
 	H.equip_in_one_of_slots(B, slots , qdel_on_fail = TRUE)
 
+/datum/quirk/linguist
+	name = "Linguist"
+	desc = "Although you don't know every language, you have spent your interest into languages. You can recognise linguistic features for each language."
```

Comment:
Although you don't know every language, your intense interest in languages allows you to recognize the features of most languages.